config/database.yml.example

```diff
@@ -23,3 +25,72 @@ production:
   database: db/production.sqlite3
   pool: 5
   timeout: 5000
+
+#   PostgreSQL Configuration:
+#   gem install pg
+#
+#   Ensure the PostgreSQL 'pg' gem is defined in your Gemfile
+#   gem 'pg'
+#   Update with your credentials below for username and password.
```

Comment:
Syntax-wise this should be:
'Update your credentials for username and password'